src/api/app/controllers/webui/comments_controller.rb

```diff
@@ -32,6 +32,23 @@ def destroy
     render(partial: 'webui/comment/comment_list', locals: { commentable: @commentable }, status: status)
   end
 
+  def update
+    comment = Comment.find(params[:id])
+    authorize comment, :update?
+
+    comment.assign_attributes(permitted_params)
+    @commentable = comment.commentable
```

Comment:
If I'm not wrong, this `@commentable` instance variable is not needed, because you are not using it in the view and you are passing the commentable as a local variable [here](https://github.com/openSUSE/open-build-service/pull/10148/files#diff-1a8fabf68e383a44d297342d83cfec32R49).

Reply:
It's true, for some reason I thought that the instance variable was used somewhere. Nice one noticing it :1st_place_medal: